packages/chat-core/src/index.ts

```diff
@@ -583,6 +605,19 @@ export interface ConversationHandler {
    */
   sendIntent: (intentId: string, context?: Context) => void;
 
+  /**
+   * Obtain LiveKit credentials to run the experience in voice.
+   * @internal
+   * @returns LiveKit credentials in promise form
+   */
+  getLiveKitCredentials: () => Promise<LiveKitCredentials>;
+
+  /**
+   * Terminate LiveKit call
+   * @internal
+   */
+  terminateLiveKitCall: () => Promise<unknown>;
```

Comment:
```suggestion
  terminateLiveKitCall: () => Promise<void>;
```

it's a bit weird to have `unknown` in our own API.

Reply:
In this case I find it to be the most honest way to type it, I'm trying to say 'something comes in the response, do not rely on it'.

Reply:
yea